<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShadowConfiguraionUI</class>
 <widget class="QWidget" name="ShadowConfiguraionUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>180</width>
    <height>112</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Inner Color:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>innerColor</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="KColorButton" name="innerColor"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Outer Color:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>outerColor</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="KColorButton" name="outerColor">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QCheckBox" name="useOuterColor">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Vertical Offset:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>verticalOffset</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Size: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>shadowSize</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KIntSpinBox" name="shadowSize">
     <property name="suffix">
      <string>px</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KIntSpinBox" name="verticalOffset">
     <property name="suffix">
      <string>px</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>shadowSize</tabstop>
  <tabstop>verticalOffset</tabstop>
  <tabstop>innerColor</tabstop>
  <tabstop>outerColor</tabstop>
  <tabstop>useOuterColor</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
